### PR TITLE
LPS-197321 Change Actions FF to the dataset manager FF

### PR DESCRIPTION
**Pull request by kresimir-coko**
On hold until https://github.com/liferay-frontend/liferay-portal/pull/3673 gets merged

### CODE REVIEW

**Review by markocikos — APPROVED** [review]
@kresimir-coko LGTM! :sparkles:

**Comment by markocikos** [comment]
@kresimir-coko The change in code is fine, but `LPS-192281` is not. This is the `Task˙ LPS. You should create a subtask within a task, and send a code under that LPS. You can also set subtask to `In Peer Review` status.

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSView.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ if (Liferay.FeatureFlags['LPS-188645']) {
 	];
 }
 
-if (Liferay.FeatureFlags['LPS-192282']) {
+if (Liferay.FeatureFlags['LPS-164563']) {
```
Comment (dsanz) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSView.tsx:47]: We don't need this as LPS-164563 FF is gating the entire Data Set Manager, so basically we just need to unconditionally add `Actions` to the nav bar items

Comment (kresimir-coko) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSView.tsx:47]: I tried looking into that and couldn't verify, so went with the safe variant. Will fix! Thanks @dsanz

**Review by dsanz — APPROVED** [review]
LGTM ;) :tada:

**Comment by dsanz** [comment]
ci:forward

**Comment by dsanz** [comment]
ci:reevaluate:1287401_7727

**Comment by andreluis2022** [comment]
I'm taking a look at the flaws.

**Comment by andreluis2022** [comment]
Hey @kresimir-coko, the failures seem to be unrelated to your PR, according to this [comparison](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=2&testrayRunIdB=2431951152&testrayRunIdA=2432031721&statusB=3)

cc: @mateusralv

**Comment by markocikos** [comment]
ci:forward:force

**Pull request closed without merging**